plugins/mcreator-core/procedures/explode.json

```diff
@@ -19,6 +19,24 @@
       "type": "input_value",
       "name": "z",
       "check": "Number"
+    },
+    {
+      "type": "field_dropdown",
+      "name": "mode",
+      "options": [
+        [
```

Comment:
Put BREAK the first on the list so existing procedure blocks will have the proper default value selected, also better default value in general.

Reply:
I put them in the order they are present in MC code.
Also, this default value is declared in `toolbox_init` xml:
```json
53      "<field name=\"mode\">BREAK</field>"
```

Reply:
This default value will not be applied to existing procedure blocks ...

Reply:
>This default value will not be applied to existing procedure blocks ...

That's why I added FV converter.

Reply:
But there is no need to. FV converters are slow and there is another alternative I provided you which is a common way to do this to make this work and keep things performance-wise ok

Reply:
Then I should give new 'UI names' to explosion modes.